modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -116,7 +116,7 @@ exports.oauthCallback = function (strategy) {
           return res.redirect('/authentication/signin');
         }
 
-        return res.redirect(redirectURL || sessionRedirectURL || '/');
```

Comment:
seems can not simply update: 
_return res.redirect(redirectURL || sessionRedirectURL || '/');_
to:
_return res.redirect(sessionRedirectURL || '/');_

because before **oauthCallback**  function called, there will be another function **saveOAuthUserProfile** execute before it and have the following code snippet to pass value to redirectURL(changed to info): 
      `user.save(function (err) {
        return done(err, user, '/settings/accounts');
      });`
so, if pass value to redirectURL, the value will be right, if do not pass anything, the value will become an object.

for your reference, thanks.